docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -65,40 +70,24 @@ This method doesn't use the command line and can be accomplished using commands
+1. Verify that your add-in is installed. For example, if it has an add-in command, it should appear on either the ribbon or the context menu. If it's a task pane add-in that has no add-in commands, the task pane should appear.
 
 > [!NOTE]
 > To test your Office Add-in with Microsoft Edge with the original WebView (EdgeHTML), an additional configuration step is required. In a Windows Command Prompt, run the following line: `npx office-addin-dev-settings appcontainer EdgeWebView --loopback --yes`. This is not required when Office is using the Chromium-based Edge WebView2. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
 
 [!INCLUDE[Office settings tool not supported on Mac](../includes/tool-nonsupport-mac-note.md)]
 
-## Sideload an Office Add-in
+## Sideload an add-in to Microsoft 365
 
 1. Sign in to your Microsoft 365 account.
 
 1. Open the App Launcher on the left end of the toolbar and select **Excel**, **PowerPoint**, or **Word**, and then create a new document.
 
-1. Steps 3 - 6 are the same as in the preceding section **Sideload an Office Add-in in Office on the web**.
-
-## Sideload an add-in when using Visual Studio
-
-If you're using Visual Studio to develop your add-in, the process to sideload is similar to manual sideloading to the web. The only difference is that you must update the value of the **SourceURL** element in your manifest to include the full URL where the add-in is deployed.
-
-> [!NOTE]
-> Although you can sideload add-ins from Visual Studio to Office on the web, you cannot debug them from Visual Studio. To debug you will need to use the browser debugging tools. For more information, see [Debug add-ins in Office on the web](debug-add-ins-in-office-online.md).
-
-1. In Visual Studio, show the **Properties** window by choosing **View** > **Properties Window**.
-1. In the **Solution Explorer**, select the web project. This will display properties for the project in the **Properties** window.
-1. In the Properties window, copy the **SSL URL**.
-1. In the add-in project, open the manifest XML file. Be sure you are editing the source XML. For some project types Visual Studio will open a visual view of the XML which will not work for the next step.
-1. Search and replace all instances of **~remoteAppUrl/** with the SSL URL you just copied. You will see several replacements depending on the project type, and the new URLs will appear similar to `https://localhost:44300/Home.html`.
-1. Save the XML file.
-1. Right click the web project and choose **Debug** > **Start new instance**. This will run the web project without launching Office.
-1. From Office on the web, sideload the add-in using steps previously described in [Sideload an Office Add-in in Office on the web](#sideload-an-office-add-in-in-office-on-the-web).
+1. Steps 3 - 6 are the same as in the section [Manually sideload an add-in to Office on the web](#manually-sideload-an-add-in-to-office-on-the-web).
```

Comment:
I read this and kind of thought I needed to follow steps 3-6 of the "Manually sideload an add-ins to Office on the web" section.

If you accept my suggestion for that section, then this can say something like:
```suggestion
1. Follow steps 2 - 5 of the [Manually sideload an add-in to Office on the web](#manually-sideload-an-add-in-to-office-on-the-web) section.
```